api/api.go

```diff
@@ -231,7 +231,7 @@ func New(
 		password:              password,
 		volumePath:            defaultVolumesPath,
 		volumePathPermissions: defaultVolumesPathPermissions,
-		unresolvableHosts:     defaultUnresolvableHosts,
+		unresolvableHosts:     defaultIgnoreUnresolvableHosts,
```

Comment:
rename unresolvableHosts to ignoreUnresolvableHosts. Also update in places where this is referred.

Reply:
renamed to ignoreUnresolvableHosts

Reply:
but if you see other var's they are not exactly same as mentioned in values file. "ignoreUnresolvableHosts" param is already mentioned in values file so would it be good to have same var here too

Reply:
Basically here I am asking to provide a meaningful name for the variable. unresolvableHosts is different from ignoreUnresolvableHosts. It is ok to have same names in values and in goisilon, as long as they are meaningful.